src/test/javascript/spec/component/programming-exercise/programming-exercise-test-schedule-date-picker.component.spec.ts

```diff
@@ -0,0 +1,94 @@
+import dayjs from 'dayjs/esm';
+import { NgModel } from '@angular/forms';
+import { ProgrammingExerciseTestScheduleDatePickerComponent } from 'app/exercises/programming/shared/lifecycle/programming-exercise-test-schedule-date-picker.component';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ArtemisTestModule } from '../../test.module';
+import { OwlDateTimeModule, OwlNativeDateTimeModule } from '@danielmoncada/angular-datetime-picker';
+import { MockDirective, MockPipe } from 'ng-mocks';
+import { ArtemisDatePipe } from 'app/shared/pipes/artemis-date.pipe';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import { TranslateDirective } from 'app/shared/language/translate.directive';
+
+describe('ProgrammingExerciseTestScheduleDatePickerComponent', () => {
+    let comp: ProgrammingExerciseTestScheduleDatePickerComponent;
+    let fixture: ComponentFixture<ProgrammingExerciseTestScheduleDatePickerComponent>;
+
+    const selectedDate = dayjs().add(5, 'days').toDate();
+    const startAt = dayjs().add(6, 'days');
+    const min = dayjs();
+    const max = dayjs().add(7, 'days');
+    const label = '';
+    const tooltipText = '';
+    const readOnly = false;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule, OwlNativeDateTimeModule, OwlDateTimeModule],
```

Comment:
```suggestion
            imports: [ArtemisTestModule, MockModule(OwlDateTimeModule)],
```
I think this should do the job as well

Reply:
Maybe. You can give it a try. It was actually mocked a while ago. When I bumped Angular to v13, I had to replace the mocked date time module with the real one, otherwise it didn't work anymore.

Reply:
In my case it works, so the module is mocked now.